assets/src/application/ui/layout/entityList/filterBar/style.scss

```diff
@@ -68,6 +40,40 @@
 		}
 	}
 
+	&__btn {
+		&.esprs-button.ant-btn {
+			&#{$filter-btn} {
```

Comment:
this was used to increase specificity, more info: https://www.tobymackenzie.com/blog/2016/05/10/duplicate-selectors-increase-specificity/

Reply:
that looks like a button within another button ?!?!?! is that right??

would this not be equivalent to:

```css
		&.esprs-button.ant-btn {
			.esprs-button.ant-btn.ee-entity-list-filter-bar__btn {
```

Reply:
based on my understanding, this would be the same:
```
&.esprs-button.ant-btn {
			&.esprs-button.ant-btn.ee-entity-list-filter-bar__btn {
```

Reply:
we need to find a better solution to fight antd specificity, because even if this duplication works I am not :100: happy with this approach since it's a bit convoluted

Reply:
oh right you had another `&` in there... but even so, that would essentially evaluate to the equivalent of:

```css
.esprs-button.ant-btn.esprs-button.ant-btn.ee-entity-list-filter-bar__btn
```

which is just a bit redundant....
since `$filter-btn` equals '.esprs-button.ant-btn.ee-entity-list-filter-bar__btn';

then can't this just be:

```css
	&__btn {
		&#{$filter-btn} {
```

???

Reply:
yes, that is simpler, thx:
https://github.com/eventespresso/event-espresso-core/pull/2578/commits/2e6e34df811e691238954098c128353a7a2c71cd